docs-src/concepts/what-is-the-temporal-cli.md

```diff
@@ -28,3 +21,135 @@ Workflow, show Workflow History, and Signal Workflow).
+| -------------------------------------------- | --------------------------------------------------------------------------------- | ------------------------------- |
+| `TEMPORAL_CLI_ADDRESS`                       | The host and port (formatted as host:port) for the Temporal Frontend Service.     | --address                       |
+| `TEMPORAL_CLI_CODEC_AUTH`                    | Sets the authorization header on requests to the Codec Server.                    | --codec-auth                    |
+| `TEMPORAL_CLI_CODEC_ENDPOINT`                | Endpoint for a remote Codec Server.                                               | --codec-endpoint                |
+| `TEMPORAL_CONTEXT_TIMEOUT`                   | An optional timeout for the context of an RPC call (in seconds). Default value: 5 | --context-timeout               |
+| `TEMPORAL_CLI_NAMESPACE`                     | Identifies a Namespace in the Temporal Workflow. Default value: "default".        | --namespace                     |
+| `TEMPORAL_CLI_TLS_CA`                        | Path to server CA certificate.                                                    | --tls-ca-path                   |
+| `TEMPORAL_CLI_TLS_CERT`                      | Path to x509 certificate.                                                         | --tls-cert-path                 |
+| `TEMPORAL_CLI_TLS_DISABLE_HOST_VERIFICATION` | Disables TLS host name verification if already enabled. Default value: false.     | --tls-disable-host-verification |
+| `TEMPORAL_CLI_TLS_KEY`                       | Path to private certificate key.                                                  | --tls-key-path                  |
+| `TEMPORAL_CLI_TLS_SERVER_NAME`               | Provides an override for the target TLS server name.                              | --tls-server-name               |
+
+## Auto-completion
+
+The Temporal CLI has the capability to auto-complete commands.
+
+Running `temporal completion SHELL` outputs the related completion SHELL code.
+
+### zsh auto-completion
+
+<!-- TODO: add more information about zsh to make comparable to bash section -->
+
+Add the following code snippet to your `~/.zshrc` file:
+
+```sh
+source <(temporal completion zsh)
+```
+
+If you're running auto-completion from the terminal, run the following command:
+
+```sh
+echo 'source <(temporal completion zsh)' >> ~/.zshrc
+```
+
+After setting the variable, run the following command:
+
+`source ~/.zshrc`.
+
+### Bash auto-completion
+
+Bash auto-completion relies on `bash-completion`.
+
+Install the software with the steps provided in the [bash-completion README](https://github.com/scop/bash-completion#installation), or use your preferred package manager on your operating system.
+
+To view different bash versions, go [here](https://repology.org/project/bash-completion/versions).
```

Comment:
Should show how to install the completion like we do with zsh completion.


> To view different bash versions, go [here](https://repology.org/project/bash-completion/versions).

These aren't different bash versions.